features/generators.feature

```diff
@@ -51,4 +57,4 @@ Feature:
       end
       """
     And I run `bundle exec rails generate model User name:string` with a clean environment
-    Then the file "test/factories.rb" should contain "factory :user do"
+    Then the file "test/factories/users.rb" should not contain "factory :user do"
```

Comment:
Not strictly related to this PR, but this expectation didn't match Scenario description.